packages/shared/components/Search/SearchPagination.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Flex } from 'design';
+import { StyledPanel } from 'design/DataTable/StyledTable';
+import { StyledArrowBtn } from 'design/DataTable/Pager/StyledPager';
```

Comment:
This file (and `SearchPanel`) were just abstracted away from the table used in `NewRequest` so that it could be reused here. If I'm going to change the implementation then these probably wouldn't need to exist anymore